Source/sourcekitten/Doc.swift

```diff
@@ -0,0 +1,71 @@
+import ArgumentParser
+import Foundation
+import SourceKittenFramework
+
+extension SourceKitten {
+    struct Doc: ParsableCommand {
+        static let configuration = CommandConfiguration(abstract: "Print Swift or Objective-C docs as JSON")
+
+        @Option(help: "Only document one file")
+        var singleFile: Bool = false
+        @Option(help: "Name of Swift module to document (can't be used with `--single-file`)")
+        var moduleName: String = ""
+        @Option(help: "Document a Swift Package Manager module")
+        var spm: Bool = false
```

Comment:
The `singleFile`, `spm`, and `objc` properties need to be `@Flag` (or something like that, not used the library) otherwise it tries to eat the next token as the value, I think the same problem in other commands (eg. `complete --spm-module`).

Reply:
Good catch! Fixed in 186c4f6.